src/test/java/io/stargate/sgv2/jsonapi/testresource/DseTestResource.java

```diff
@@ -6,20 +6,29 @@
 
 public class DseTestResource extends StargateTestResource {
   // set default props if not set, so we launch DSE
-  // this is only needed for test from the IDE
+  // this is only needed for tests run from the IDE
   public DseTestResource() {
     super();
 
     if (null == System.getProperty("testing.containers.cassandra-image")) {
+      // 14-Mar-2025, tatu: Change from custom "dse-next" to the official DSE image
+      //  even for IDE tests
       System.setProperty(
-          "testing.containers.cassandra-image", "stargateio/dse-next:4.0.11-591d171ac9c9");
+          "testing.containers.cassandra-image",
+          // "stargateio/dse-next:4.0.11-591d171ac9c9"
+          "datastax/dse-server:6.9.7");
+      // MUST set this to get DS_LICENSE env var set
+      System.setProperty("testing.containers.cluster-dse", "true");
     }
 
+    // 14-Mar-2025, tatu: We no longer run Stargate Coordinator for ITs, left this here
```

Comment:
maybe better to just remove, we don't intend to go back

Reply:
Ok, will just remove.